### ISSUE TITLE
[Screen Reader - Windows Powershell]: An element of the given ControlType must support the Selection pattern.

### ISSUE THREAD

**Issue by Chandresh1007** [microsoft/terminal#8482]
**User Experience:**
When standard controls from accessible technologies are not used and if the user interface elements are not used according to specification, Assistive Technologies (AT) will not be able to gather information about, activate(or set) and keep up to date on the status of user interface controls in the content.

**Test Environment:**
OS: Win 10 2004 Build (20262.1010)
App: Windows Terminal
App version: 1.4.3243.0
Tool: Accessibility Insights for Windows

**Repro Steps:**
1. Open Windows terminal app.
2. Press "Ctrl+Shift+T" to open multiple tabs.
3. Now run accessibility insight tool on the application.
4. Observe the issue.

**Actual Result:**
An element of the given ControlType does not support the Selection pattern.
**Narrator does not announce the state of the Tab as 'selected'**.

**Expected Result:**
An element of the given ControlType must support the Selection pattern.
**Narrator should announce the state of selected tab as 'Selected'**.

**Note:** Same issue repro with NVDA.

**MAS Reference:**
**MAS4.1.2**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc=%7B248054a6-5e68-4771-9e1e-242fb5025730%7D&cid=e71aec14-4173-423e-aecb-2472bbc751e8

![MAS4 1 2_AI_Selection pattern_Details](https://user-images.githubusercontent.com/53220700/101023137-57d7e580-3598-11eb-9370-db4f965bb49a.png)

![MAS4 1 2_AI_Selection pattern_how to fix](https://user-images.githubusercontent.com/53220700/101023147-59a1a900-3598-11eb-8b10-ecc85f34d7f1.png)

[MAS4.1.2_State does not get announced.zip](https://github.com/microsoft/terminal/files/5636294/MAS4.1.2_State.does.not.get.announced.zip)

**Comment by carlos-zamora** [comment]
This is the TabView control (type `Microsoft.UI.Xaml.Controls.TabView`). Hosts the list of tabs and the split button:
![UIA Tree](https://user-images.githubusercontent.com/11050425/101068672-dce9dd00-354d-11eb-98be-0967da70ab11.png)

This should probably be done by the XAML team so that all TabView's just get this functionality for free. I wonder if this is related to #8464?

SelectionPatternIdentifiers docs ([WinUI3](https://docs.microsoft.com/en-us/windows/winui/api/microsoft.ui.xaml.automation.selectionpatternidentifiers?view=winui-3.0-preview) & [WinRT](https://docs.microsoft.com/en-us/uwp/api/Windows.UI.Xaml.Automation.SelectionPatternIdentifiers?view=winrt-19041)).

**Comment by DHowett** [comment]
@Chandresh1007 Cannot reproduce. Can you validate?

Windows Terminal 1.5. Stable version.

![image](https://user-images.githubusercontent.com/189190/108916933-d23a6380-75e3-11eb-97f3-29f061194110.png)

"There were no failures." for the tab element.

**Comment by Priyanka-Chauhan123** [comment]
Issue is fixed. Verified on following Test Environment:
OS: Win 10 2004 Build (21301.1010)
App: Windows Terminal
App version: 1.5.10411.0
Tool: Accessibility Insights for Windows
Hence, closing the bug.
